app/controllers/admin/transactions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Admin::TransactionsController < Admin::AdminController
```

Comment:
Should be namespaced as `Admin::Accounting::`. I think we will end up with several controllers here and it could get awkward.